app/src/services/enquiry.ts

```diff
@@ -80,6 +80,26 @@ const service = {
     }
   },
 
+  /**
+   * @function getEnquiriesFromActivityIds
+   * Gets list of enquiries from their activityIds
+   * @param {string[]} activityIds Activity ID
+   * @returns {Promise<Enquiry | null>} The result of running the findMany operation
+   */
+  getEnquiriesFromActivityIds: async (activityIds: Array<string>) => {
+    try {
+      const result = await prisma.enquiry.findMany({
+        where: {
+          activity_id: { in: activityIds }
+        }
+      });
+
+      return result.map((x) => enquiry.fromPrismaModel(x));
+    } catch (e: unknown) {
+      throw e;
+    }
+  },
+
```

Comment:
Consider turning this into a generic `searchEnquiries`. Reference the search function in the submission service.

Reply:
done!